root/static/scripts/edit/MB/Control/Bubble.js

```diff
@@ -273,7 +273,7 @@ function bubbleControlHandler(event) {
                 bubble.hide(false);
             }
         }
-        return;
+        return true;
```

Comment:
For event handlers I think `return undefined` makes more sense, since `false` is the only value that isn't ignored. `return true` could be slightly misleading since it might imply to a reader that a `true` value is used somehow.